examples/webgl_loader_obj.html

```diff
@@ -93,7 +93,7 @@
 				};
 
 				var loader = new THREE.OBJLoader( manager );
-				loader.load( 'obj/male02/male02.obj', function ( object ) {
+				loader.load( 'obj/pointCloudBox/pointCloudBox.obj', function ( object ) {
```

Comment:
I would not change the example. Just adding the `OBJ` file for testing should be sufficient for this feature.